<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventMedicalDiagnosisPageWidget</class>
 <widget class="QWidget" name="EventMedicalDiagnosisPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Диагноз</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrEventMedicalDiagnosisPage">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <widget class="QWidget" name="scrEventMedicalDiagnosisPageInterior">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>613</width>
        <height>580</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CEventMedicalDiagnosisInDocTableView" name="tblEventMedicalDiagnosis">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CEventMedicalDiagnosisInDocTableView</class>
   <extends>QTableView</extends>
   <header>Events/EventMedicalDiagnosisInDocTableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrEventMedicalDiagnosisPage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
